dart/examples/web_example/README.md

```diff
@@ -0,0 +1,15 @@
+# Sentry Dart : web example
+
+```dart
+pub get
+webdev serve --release
```

Comment:
What's `webdev`? should we propose installing that? 

In the Flutter Web example I used `python3` which should be installed on macOS and Linux OOTB: https://github.com/getsentry/sentry-dart/blob/main/flutter/example/run.sh#L54

Not true for Windows though. So maybe a comment:
```suggestion
# Serve the static files with your preferred dev static site server
webdev serve --release
```

Reply:
the project needs to be build to be served from a static site server (`webdev build ...`) .
`webdev serve` is simplest/fastest way to run this example, so I just added a link to the `webdev` page.